Makefile

```diff
@@ -25,7 +25,7 @@ MOD_FILES = util_rot.mod globalvar.mod parameters.mod \
 	boundary_rot.mod solver_rot.mod matrix_rot.mod initial_rot.mod  procedures.mod
 
 #FC = gfortran
```

Comment:
Change to the compiler choice. It would be nice to be more specific on the compiler options, but for right now, the `h5pfc` wrapper does the job.